quilt/test/test_command.py

```diff
@@ -149,7 +149,7 @@ def _mock_logs_list(self, owner, package, pkg_hash):
         logs_url = "%s/api/log/%s/%s/" % (command.QUILT_PKG_URL, owner, package)
         resp = dict(logs=[dict(
             hash=pkg_hash,
-            created=str(datetime.now()),
+            created=str(time.time()),
```

Comment:
The server will be returning a `float`, so no need to `str` it here. (Unless you think floats in JSON are a bad idea; they do lose some precision, just not enough to matter in our case.)